tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -604,7 +597,13 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
                 response should be(activation.withoutLogs.toExtendedJson)
             }
 
-            // repeat invoke, get only result back
+            // repeat invoke, get only result back (result option implies blocking invocation)
```

Comment:
yes, the PG test failed this test case . Changed it back to the origin test case behaviour.